pandas/core/base.py

```diff
@@ -231,8 +237,11 @@ class AccessorProperty(object):
     """Descriptor for implementing accessor properties like Series.str
     """
 
-    def __init__(self, accessor_cls, construct_accessor):
+    def __init__(self, accessor_cls, construct_accessor=None):
         self.accessor_cls = accessor_cls
+        if construct_accessor is None:
```

Comment:
``self.construct_accessor = construct_accessor or accessor_cls._make_accessor`` is idiomatic